src/main/java/org/isf/ward/model/Ward.java

```diff
@@ -314,7 +318,26 @@ public int hashCode() {
 		}
 		return this.hashCode;
 	}
-	
-	
 
+	public static class WardCodeComparator implements Comparator<Ward> {
+
+		@Override
+		public int compare(Ward object1, Ward object2) {
+			if (object2.getCode() == null) {
+				return -1;
+			}
+			return object1.getCode().compareTo(object2.getCode());
+		}
+	}
+
+	public static class WardDescriptionComparator implements Comparator<Ward> {
+
+		@Override
+		public int compare(Ward object1, Ward object2) {
+			if (object2.getDescription() == null) {
+				return -1;
+			}
+			return object1.getDescription().compareTo(object2.getDescription());
```

Comment:
```suggestion
			return object2.getDescription().compareTo(object1.getDescription());
```
Same comment as above